src/classMembersNameRule.ts

```diff
@@ -47,39 +45,34 @@ interface RuleOptions {
 }
 
 namespace FormatHelpers {
-    export function getLeadingUnderscore(text: string): string {
-        let result: string = "";
-
-        for (let i = 0; i < text.length; i++) {
-            if (text[i] !== "_") {
-                break;
-            }
-
-            result += text[i];
+    export function changeFormat(format: Format, text: string, prefix?: string): string {
+        let textWithoutPrefix: string;
+        if (prefix != null && text.startsWith(prefix)) {
+            textWithoutPrefix = text.substring(prefix.length, text.length);
+        } else {
+            textWithoutPrefix = text;
         }
 
-        return result;
-    }
-
-    export function changeFormat(format: Format, text: string, leadingUnderscore?: boolean): string {
-        const leadingUnderscoreText: string = leadingUnderscore ? getLeadingUnderscore(text) : "";
+        if (prefix == null) {

```

Comment:
Maybe use default value `""` in function parameters.

Also would prevent from checking `prefix == null` multiple times.

Reply:
Fixed.